src/test/suite/editors/playgroundController.test.ts

```diff
@@ -116,5 +119,29 @@ suite('Playground Controller Test Suite', () => {
         }
       );
     });
+
+    test('show a confirmation message before running commands in a playground', (done) => {
```

Comment:
nice - I think this test also covers the case where a user doesn't hit `yes`.

Reply:
I replaced this test with two, where we can check both states when prompt should be shown and when not. Could you please take a look again.